tests/javax/microedition/lcdui/TestGraphicsClipping.java

```diff
@@ -105,6 +106,21 @@ public void test(TestHarness th) {
                         image.getRGB(clippedData, 0, width, 0, 0, width, height);
 
                         th.check(verifyClip(unclippedData, clippedData, cx, cy, cw, ch));
+
+                        // Ensure that the various getClip* calls return
+                        // the correct values.
+                        th.check(g.getClipX(), cx);
+                        th.check(g.getClipY(), cy);
+                        th.check(g.getClipWidth(), cw);
+                        th.check(g.getClipHeight(), ch);
+
+                        // getClip returns a pair of coordinates rather than
```

Comment:
> nit: "a pair of" is confusing here since it returns 4 values

Indeed. It's technically correct, since a coordinate is a pair of values, but the "pair of pairs" implication is confusing. I've replaced it in ef8f471 with: "getClip returns the coordinates of the top-left and bottom-right corners rather than the top-left coordinate and a width/height."